recipes/pnab/bld.bat

```diff
@@ -0,0 +1,14 @@
+setlocal EnableDelayedExpansion
+
+CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86_amd64
```

Comment:
@alenaizan I'm not a windows expert but I think conda-build handles that for you.
@isuruf should we remove this line and pin vc14?

Reply:
Yes, this is unneeded.

Reply:
Fixed.